CorsixTH/Lua/graphics.lua

```diff
@@ -336,10 +336,10 @@ function Graphics:onChangeLanguage()
   self.load_info = {} -- Any newly made objects are temporary, and shouldn't
                       -- remember reload information (also avoids insertions
                       -- into a table being iterated over).
-  for object, load_info in pairs(load_info) do
+  for object, load_i in pairs(load_info) do
```

Comment:
Abbreviating the name throws away information, in this case "pairs(load_info)" looks wrong (iterating over a non-plural value looks weird.
I would suggest you rename that variable to "load_infos" instead to express it's a container with many values (and the self.load_info is thus also wrong, but better not touch that, it could be used in many places)